src/components/containers/Modal.tsx

```diff
@@ -205,23 +240,43 @@ export default function Modal({
               flex: 1,
               backgroundColor: componentStyle.header.primaryBackgroundColor,
               transition: 'all ease .25s',
+              display: "flex",
+              flexDirection: "column",
+              justifyContent: "flex-end",
             }}
           >
-            <H3
+            <div
               ref={observe}
-              color='white'
-              additionalStyles={{
-                margin: 0,
-                padding: 0,
+              style={{
                 paddingRight: 25,
                 paddingLeft: 25,
-                top: 35,
-                position: 'relative',
+                top: titleTopOffset,
+                position: "relative",
               }}
-              useTheme={false}
             >
-              {title}
-            </H3>
+              <TitleComponent
+                useTheme={false}
```

Comment:
curious why is the `useTheme` prop here? TitleComponent is just basically the header font definition on 157, correct?

Reply:
That's a good question. I honestly just copied it over from the existing header element (old line 221), and if we do use the theme then the header looks wrong.